docs/using/polar-syntax.rst

```diff
@@ -279,14 +279,36 @@ Dictionary Key Access
 ^^^^^^^^^^^^^^^^^^^^^
 
 The dot ``.`` operator can be used to access the value associated with
-a key in a dictionary or class instance. For example, the rule::
+a key in a dictionary. For example::
 
-  first_name(dict, x) if
-    dict = new Person() and
-    x = dict.first_name;
+    dict = { hello: "world" } and
+    dict.hello = "world"
+
+A string stored in a variable can be used as the key in a dot lookup using the following syntax::
+
+    dict = { hello: "world" } and
+    key = "hello" and
+    dict.(key) = "world"
+
+.. _application-methods:
+
+Application Method Calls
```

Comment:
Maybe a more generic title here since this is talking about both field access and method calls?